src/worker/helpers.go

```diff
@@ -170,7 +170,7 @@ func initOLDir(olPath string, dockerBaseImage string, newBase bool) (err error)
 // main error scenarios:
 // 1. PID exists, but process cannot be killed (worker probably died unexpectedly)
 // 2. The cleanup is taking too long (maybe the timeout is insufficient, or there is a deadlock)
-func stopOL(_ string) error {
+func stopOL(olPath string) error {
```

Comment:
let's add comments below indicating each of the scenarios we're handling:
1. worker already stopped
2. clean shutdown (via kill signal)
3. dirty shutdown (there's a worker.pid, but no corresponding process)